src/App.tsx

```diff
@@ -1,29 +1,53 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
```

Comment:
Consider moving the `items` array into the `App` component if it's not used elsewhere. This would encapsulate it within the component that uses it, which is generally a good practice for maintainability.